org.eclipse.corrosion/src/org/eclipse/corrosion/CorrosionPreferenceInitializer.java

```diff
@@ -17,13 +18,15 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 
+import org.eclipse.core.runtime.Platform;
 import org.eclipse.core.runtime.preferences.AbstractPreferenceInitializer;
 import org.eclipse.jface.preference.IPreferenceStore;
 
 public class CorrosionPreferenceInitializer extends AbstractPreferenceInitializer {
 
 	private static final IPreferenceStore STORE = CorrosionPlugin.getDefault().getPreferenceStore();
-	private static final String USER_HOME_PROPERTY = "user.home"; //$NON-NLS-1$
+	private static final String CARGO_DEFAULT_HOME = System.getProperty("user.home") + "/.cargo/bin/"; //$NON-NLS-1$ //$NON-NLS-2$
+	private static final boolean IS_WINDOWS = Platform.getOS().startsWith("win"); //$NON-NLS-1$
```

Comment:
Should be Platform.OS_WIN32.equals(Platform.getOS())

Reply:
Being fixed in #100